docs/adr/0001-daily-active-use-server-side-metrics-glean.md

```diff
@@ -0,0 +1,113 @@
+# Measuring Server-Side Daily Active Use (DAU) With Glean
+
+* Status: proposed
+* Deciders: Taddes Korris, David Durst, JR Conlin, Phillip Jenvey
+* Date: 2024-10-09
+
+Technical Story: 
+[Jira Epic Url](https://mozilla-hub.atlassian.net/browse/SYNC-4389)
+
+
+## Context and Problem Statement
+
+There is a requirement to move away from the current measurement of Sync Daily Active Use (DAU), which measures usage via FxA/Mozilla Accounts.
```

Comment:
Might be better to say that the requirement is for each service to be able to measure their use directly. FxA will still be able to measure this, and we should be able to compare our measurement with theirs.